lib/srv/db/ca.go

```diff
@@ -205,54 +317,98 @@ func (d *realDownloader) Download(ctx context.Context, database types.Database,
 		} else if strings.HasSuffix(azureCAURLDigiCert, hint) {
 			return d.downloadFromURL(azureCAURLDigiCert)
 		}
-		return nil, trace.BadParameter("unknown Azure CA %q", hint)
+		return nil, nil, trace.BadParameter("unknown Azure CA %q", hint)
 	case types.DatabaseTypeAWSKeyspaces:
 		return d.downloadFromURL(amazonKeyspacesCAURL)
 	}
-	return nil, trace.BadParameter("%v doesn't support automatic CA download", database)
+	return nil, nil, trace.BadParameter("%v doesn't support automatic CA download", database)
+}
+
+// GetVersion returns the CA version for the provided database.
+func (d *realDownloader) GetVersion(ctx context.Context, database types.Database, hint string) ([]byte, error) {
+	switch database.GetType() {
```

Comment:
We cannot return the version without "downloading" (the API returns everything). So it will return not implemented and be handled by the `Download` flow.